exhibitor-appliance.yaml

```diff
@@ -69,10 +71,18 @@ Resources:
       - InstancePort: 8181
         LoadBalancerPort: 80
         Protocol: TCP
-      LoadBalancerName: "exhibitor-{{Arguments.version}}"
+      - InstancePort: 2181
+        LoadBalancerPort: 2181
+        Protocol: TCP
+      - InstancePort: 2888
+        LoadBalancerPort: 2888
+        Protocol: TCP
+      - InstancePort: 3888
+        LoadBalancerPort: 3888
+        Protocol: TCP
       SecurityGroups:
-      - "{{Arguments.SecurityGroup}}"
-      Scheme: internal
+      - "Fn::GetAtt" : [ "ExhibitorSecGroup" , "GroupId" ]
+      Scheme: internet-facing
```

Comment:
to enable to access the webui without ssh tunnel, but yes, it is not secure enough